.github/workflows/terraform-apply-env.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       TF_VAR_cf_user: ${{ secrets.CF_USERNAME }}
       TF_VAR_cf_password: ${{ secrets.CF_PASSWORD }}
       TF_VAR_new_relic_license_key: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
+      TF_VAR_new_relic_account_id: ${{ secrets.NEW_RELIC_ACCOUNT_ID }}
```

Comment:
These values are going to need to be stored in Github Secrets so we can validate the tfplan

Reply:
👍 done!